tests/python/relay/test_ir_parser2.py

```diff
@@ -1,891 +0,0 @@
-# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Why are you removing test? If it doesnt work yet we should comment them and fix it in next pr.

Reply:
This was a duplicated file that was modified to work on new parser. I deleted the original parsers tests and replaced it with the one that has been ported to work on new parser.